packages/ibm-products/src/components/Datagrid/Datagrid/addons/CustomizeColumns/DraggableItemsList.js

```diff
@@ -41,15 +41,15 @@ export const DraggableItemsList = ({
   const visibleCols = columns
     // hide the columns without Header, e.g the sticky actions, spacer
     .filter((colDef) => {
-      return !!getColTitle(colDef);
+      return !!getNodeTextContent(colDef.Header);
```

Comment:
This is a problem as `getNodeTextContent()` always returns string. If the argument is not valid, it will return an empty string.

Reply:
Fixed [here](https://github.com/carbon-design-system/ibm-products/pull/3669/commits/941335e8092039cd8d9013d56aff5fbe15a886b0)